### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by knysh**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -80,17 +114,17 @@ All the services could be resolved from the DI container via ServiceProvider.
         {
         }
 
-        protected override ElementActionRetrier ActionRetrier => ApplicationManager.ServiceProvider.GetRequiredService<ElementActionRetrier>();
+        protected override ElementActionRetrier ActionRetrier => AqualityServices.ServiceProvider.GetService<ElementActionRetrier>();
```
Comment (mialeska) [inline README.md]: IElementActionRetrier

File: README.md [hunk 2]
```diff
@@ -80,17 +114,17 @@ All the services could be resolved from the DI container via ServiceProvider.
...
 
         protected override IApplication Application => ApplicationManager.Application;
 
-        protected override ConditionalWait ConditionalWait => ApplicationManager.ServiceProvider.GetRequiredService<ConditionalWait>();
+        protected override IConditionalWait ConditionalWait => AqualityServices.ServiceProvider.GetService<IConditionalWait>();
```
Comment (mialeska) [inline README.md]: still not merged yet

File: README.md [hunk 3]
```diff
@@ -80,17 +114,17 @@ All the services could be resolved from the DI container via ServiceProvider.
...
 
-        protected override IElementFactory Factory => ApplicationManager.ServiceProvider.GetRequiredService<IElementFactory>();
+        protected override IElementFactory Factory => AqualityServices.ServiceProvider.GetService<IElementFactory>();
 
-        protected override IElementFinder Finder => ApplicationManager.ServiceProvider.GetRequiredService<IElementFinder>();
+        protected override IElementFinder Finder => AqualityServices.ServiceProvider.GetService<IElementFinder>();
 
-        protected override LocalizationLogger LocalizationLogger => ApplicationManager.ServiceProvider.GetRequiredService<LocalizationLogger>();
+        protected override LocalizationLogger LocalizationLogger => AqualityServices.ServiceProvider.GetService<LocalizationLogger>();
```
Comment (mialeska) [inline README.md]: ILocalizedLogger?

File: README.md (hunk 2 above)
Comment (knysh) [reply README.md]: I hope we will do this.

File: README.md (hunk 1 above)
Comment (knysh) [reply README.md]: fixed

File: README.md (hunk 3 above)
Comment (knysh) [reply README.md]: fixed

**Pull request merged**